src/avatar/resolve.js

```diff
@@ -50,6 +50,7 @@ const getDefaultFallbackUrl = ({ protocol, host }) =>
 const getFallbackUrl = memoizeOne(async ({ query, protocol, host }) => {
   const { fallback } = query
   if (fallback === false) return null
+  if (fallback === 'transparent') return 'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=='
```

Comment:
Why not make possible to use data URIs as fallback image?

So this line could be turned into this:
fallback=data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==`

In this way, user have full control, WDYT?

Reply:
That’s probably a better idea, provided a transparent image is given as an example in the docs that people can quickly copy-paste!